Formula/geographiclib.rb

```diff
@@ -6,6 +6,11 @@ class Geographiclib < Formula
   license "MIT"
   head "https://github.com/geographiclib/geographiclib.git", branch: "main"
 
+  livecheck do
+    url :stable
+    regex(/^r?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^r(\d+(?:\.\d+)+)$/i)
```

Since upstream uses both `r1.2.3` and `v1.2.3` tags in the repository and the [`README` states](https://github.com/geographiclib/geographiclib#links) "specific releases are tagged as, e.g., `r1.52`, `r2.0`, etc.", there's something to be said for making the leading `r` a required part of the regex in this particular case.